src/lib/date.ts

```diff
@@ -1,8 +1,32 @@
 import moment from "moment"
 
+const formatDateStatusDisplayDate = date => {
+  const momentToUse = moment.utc(date, "YYYY-MM-DD[T]HH:mm:ss")
+  const momentDate = momentToUse.format("MMM D")
+  const momentHour = momentToUse.format("ha")
+  if (!!momentHour && !!momentDate) {
+    return ` ${momentDate} at ${momentHour}`
+  } else if (!!momentDate) {
+    return ` ${momentDate}`
```

Comment:
Is there a reason we should have the preceding whitespace here instead of below? I think it would make the code slightly more readable.

Reply:
If for some reason this returns as `null` then the rendered text will be centered without any whitespace, which still workings with the design. I.e. it'll say "Opens" under the countdown clock. Thought this was marginally better.

Reply:
Fixed this to your recommendation.